batched/dense/unit_test/Test_Batched_SerialInverseLU.hpp

```diff
@@ -143,7 +143,7 @@ void impl_test_batched_inverselu(const int N, const int BlkSize) {
   /// randomized input testing views
   AViewType a0("a0", N, BlkSize, BlkSize);
   AViewType a1("a1", N, BlkSize, BlkSize);
-  WViewType w("w", N, BlkSize * BlkSize);
+  WViewType w("w", N, BlkSize * static_cast<size_t>(BlkSize));
```

Comment:
@lucbv Why having a fix for only ```w```? How about ```a0```, ```a1```?

Reply:
That one actually does `BlkSize * BlkSize` and since both are `int`, the temporary result is also `int` but the input parameter for the view is `size_t` so there is an overflow potential before we store the result. For the other views there is no multiplication so implicit casting is working just fine.
Does that make sense?

Reply:
@Thanks. Sorry I did not see the commas in a0 and a1 :-)

Reply:
No worries!